### DISCUSSION TITLE
notice subtype definitions json files (e.g:16.json)

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#655] [Q&A] [answered]
Hello Lab,

"For each notice subtype, we define the content and structure of the form that notice authors will fill in. We provide this definition as a set of JSON files, with one file per notice subtype. Those files are available in the [eForms SDK](https://github.com/OP-TED/eForms-SDK), in the folder /notice-types, and are named as follows: <subTypeId>.json" (https://docs.ted.europa.eu/eforms/latest/notice-types/index.html)

We found that many fields missing in these notice subtype definition json files.

Example: These field are not in the 16-json but they can use optionally in eform 16.
BT-5010
BT-6140
BT-7220
BT-805
BT-717
BT-735

We are using SDK 1.7.1, but this error (see above) also occurs in SDK version 1.8.0.

Will this bug be fixed? 
If so, when is it expected to be fixed?
If not, what is the reason?

Thanks in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
These are optional fields from the Regulation Amendment and the NTDs have been progressively updated to allow for their inclusion. You will be able to find the fields you're looking for starting SDK 1.9.

**Comment by AnnaKSZ** [comment]
Thanks for the reply
Anna
